native_client/deepspeech.cc

```diff
@@ -93,33 +470,38 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
     status = ReadBinaryProto(Env::Default(), aModelPath, &mPriv->graph_def);
   }
   if (!status.ok()) {
-    mPriv->session->Close();
-    mPriv->session = NULL;
-    std::cerr << status.ToString() << std::endl;
+    (void)mPriv->session->Close();
+    mPriv->session = nullptr;
+    std::cerr << status << std::endl;
     return;
   }
 
   status = mPriv->session->Create(mPriv->graph_def);
   if (!status.ok()) {
-    mPriv->session->Close();
-    mPriv->session = NULL;
-    std::cerr << status.ToString() << std::endl;
+    (void)mPriv->session->Close();
+    mPriv->session = nullptr;
+    std::cerr << status << std::endl;
     return;
   }
 
   for (int i = 0; i < mPriv->graph_def.node_size(); ++i) {
```

Comment:
Asking out loud to myself, don't we have a better way to take care of that than iterating like that?

Reply:
Nope, I checked. The GraphDef protobuf is a list of nodes.